### PR TITLE
Completed css-brochure-form challenge.

### PR DESCRIPTION
**Pull request by thomas-tahk**
css-brochure-form

Not identical because I don't know exact numbers for paddings, margins, etc. But I believe this is somewhat close.

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  font-weight: bold;
+  color: white;
+  background-color: orangered;
+  border: 2px;
+  border-radius: 15px;
+  height: 34px;
+  width: 100px;
+  margin: 10px 0;
+  padding: 1px 0 4px;
+  float: right;
```
Comment (BrettAlbright) [inline css-brochure-form/styles.css]: I would avoid using float here, text align right should work well. float is only for wrapping text around images.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looking really good overall, a few updates are needed. 
1. Make sure the form is centered on the page, this is commonly done by wrapping the entire page in a `container` element and putting set width on it then using margin auto to center it on the page.
3.  make sure the text alignment over the inputs matches the example, the label for the final input is longer than the input
4. make sure the button alignment matches the example

**Comment by thomas-tahk** [comment]
Updated Screenshot with latest commit:
css-brochure-form-edited

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**